diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LineLengthWarnText.kt

```diff
@@ -0,0 +1,208 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings.LONG_LINE
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.LineLength
+import org.cqfn.diktat.util.lintMethod
+import org.junit.Test
+
+
+class LineLengthWarnText {
```

Comment:
You tests have a little of duplication. For example, you could have long URLs and long imports in the first positive example as well, because it is part of core logic. Please merge some of duplicated tests and instead add more diversity to you examples: property declarations, functions and constants outside of class bodies etc.